src/components/Box/__stories__/Box.stories.mdx

```diff
@@ -0,0 +1,491 @@
+import Box from "../Box";
+import {ArgsTable, Story, Canvas, Meta} from "@storybook/addon-docs";
+import {createStoryMetaSettings} from "../../../storybook/functions/create-component-story";
+import styles from "./Box.stories.module.scss";
+
+export const metaSettings = createStoryMetaSettings({
+    component: Box,
+    enumPropNamesArray: [
+        "children",
+        "disabled",
+        "border",
+        "borderTop",
+        "borderEnd",
+        "borderBottom",
+        "borderStart",
+        "borderColor",
+        "rounded",
+        "roundedTop",
+        "roundedEnd",
+        "roundedBottom",
+        "roundedStart",
+        "shadow",
+        "margin",
+        "marginX",
+        "marginY",
+        "marginTop",
+        "marginEnd",
+        "marginStart",
+        "padding",
+        "paddingX",
+        "paddingY",
+        "paddingTop",
+        "paddingEnd",
+        "paddingBottom",
+        "paddingStart",
+        "backgroundColor",
+        "text",
+    ],
+});
+
+<Meta
+    title="Layout/Box/Box"
+    component={Box}
+    argTypes={metaSettings.argTypes}
+    decorators={metaSettings.decorators}
+/>
+
+<!--- Component template -->
+
+export const boxTemplate = (args) => {
+    return <div className={styles['monday-storybook-box-wrapper']}>
+        <Box
+            border={Box.borders.DEFAULT}
+            rounded={Box.roundeds.MEDIUM}
+            {...args}></Box>
+    </div>
+}
+
+<!--- Component documentation -->
+
+# Box
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+The Box is a multi prop wrapper element.
+* Its purpose is to help scaffold compositions while using Vibe's prop keys.
+* It can be used to as a composite container for atom based compositions.
+* It can accept all Vibe style related props.
+
+<Canvas>
+    <Story name="Overview"
+           args={{children: "Box With Props"}}>
+        {boxTemplate.bind({})}
+    </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={Box}/>
+
+## Usage
+<UsageGuidelines guidelines={[
```

Comment:
let's write guidelines? important things that people need to understand before using the component.
when to use it? common use case, main features (I think shadow, vibe spacing are important... what else?

Reply:
i will push this through with Evgeny.